.github/workflows/release.yml

```diff
@@ -32,12 +33,15 @@ jobs:
       - name: Create npm package
         run: npm pack ./cucumber-tsflow
       - name: Create version info file
-        run: echo "${{ steps.set_package_version.outputs.package_version }}" > versioninfo.txt
+        run: |-
+          echo "${{ steps.set_package_version.outputs.package_version }}" > versioninfo.txt
+          jq '.releaseTag or "latest"' version.json > releasetag.txt
```

Comment:
If `jq` isn't installed by default, then this would also work (using standard shell utils):

```bash
echo -n $((cat version.json \
  | grep releaseTag || echo "\"releaseTag\": \"latest\"") \
  | head -1 \
  | awk -F: '{ print $2 }' \
  | sed 's/[",]//g' \
  | tr -d '[[:space:]]') > releasetag.txt
```